src/main/webapp/js/instructor.js

```diff
@@ -154,6 +154,22 @@ function setupFsCopyModal() {
     });
 }
 
+// For ajax failure handling
+
+/**
+ * Given an element, replaces the HTML content of the element with an error message prompting 
+ * the user to retry.
+ */
+function displayAjaxRetryMessage($element) {
```

Comment:
Moved into instructorFeedbackResults.js. Was initially thinking that the function could be reused elsewhere, to standardize error handling for the different pages that involve ajax requests by clicking on a panel header.